website/data/docs-nav-data.json

```diff
@@ -283,8 +283,7 @@
           },
           {
             "title": "Alicloud OSS",
-            "path": "configuration/storage/alicloudoss",
-            "hidden": true
+            "path": "configuration/storage/alicloudoss"
```

Comment:
I think this might be hidden because we're worried about even half supporting it, but if the code exists I think we should probably unhide it.

Reply:
It looks like we missed updating the sidebar originally and never noticed until now.  https://github.com/hashicorp/vault/pull/4783